### PR TITLE
Dlg one variable compare

### PR DESCRIPTION
**Pull request by lilyclements**
@africanmathsinitiative/developers This is ready for review

Note in order to run the compare, you have to create a model first.
There is a known issue in this dialog that you can't compare models from different variables. This is for comparing a variable with distribution X to distribution Y.

### CODE REVIEW

File: instat/dlgOneVarCompareModels.vb [hunk 1]
```diff
@@ -60,47 +76,46 @@ Public Class dlgOneVarCompareModels
...
+        If Not UcrReceiver.IsEmpty Then 'sdgOneVarCompareModels.TestOkEnabled() AndAlso Not  Then
             ucrBase.OKEnabled(True)
         Else
             ucrBase.OKEnabled(False)
         End If
     End Sub
 
-    Private Sub ucrBase_ClickReset(sender As Object, e As EventArgs) Handles ucrBase.ClickReset
-        SetDefaults()
+    Public Sub SetRCodeForControls(bReset As Boolean)
```
Comment (maxwellfundi) [inline instat/dlgOneVarCompareModels.vb]: Why do we have this sub public? Had a discussion with @dannyparsons and said that we want to keep everything in a dialog as private unless we have a reason to make them public.

File: instat/dlgOneVarCompareModels.vb [hunk 2]
```diff
@@ -60,47 +76,46 @@ Public Class dlgOneVarCompareModels
...
+        TestOKEnabled()
     End Sub
 
-    Private Sub UcrReceiver_SelectionChanged(sender As Object, e As EventArgs) Handles UcrReceiver.SelectionChanged
+    Private Sub ucrSelectorOneVarCompModels_DataFrameChanged() Handles ucrSelectorOneVarCompModels.DataFrameChanged
+        '    sdgOneVarCompareModels.DisplayChiSquare()
+    End Sub
+
+    Private Sub UcrReceiver_SelectionChanged(ucrChangedControl As ucrCore) Handles UcrReceiver.ControlContentsChanged
         If UcrReceiver.IsEmpty Then
```
Comment (maxwellfundi) [inline instat/dlgOneVarCompareModels.vb]: Wrong naming conventions here for the UcrReciver.

Comment (lilyclements) [inline instat/dlgOneVarCompareModels.vb]: Do you mean I should call it ucrReceiverCompareModels? Or that there's something in the code?
Thanks!

Comment (lilyclements) [inline instat/dlgOneVarCompareModels.vb]: I changed the name anyway, which is why there are now three files changes as oppose to one! When reviewing you just need to focus on the dlgOneVarCompareModels file

File: instat/dlgOneVarCompareModels.vb (hunk 1 above)
Comment (lilyclements) [inline instat/dlgOneVarCompareModels.vb]: Good spot. Accidentally did that! Thanks.

**Comment by lilyclements** [comment]
@maxwellfundi this is ready for a re-review

**Comment by stevekogo** [comment]
@maxwellfundi Can you test this?

**Comment by stevekogo** [comment]
@maxwellfundi can you please respond to this.

**Comment by lilyclements** [comment]
@stevekogo @maxwellfundi I don't think this is too much of a priority since the sub hasn't been fixed yet. Additionally, I'll be started on new modelling work after this summaries system is sorted and so this dialog might change

**Comment by lilyclements** [comment]
@AlexSananka did you still plan to take over on the sub dialog for this?

**Comment by AlexSananka** [comment]
if it's urgent yes I can work on it but currently working on couple of stuffs.

**Pull request closed without merging**